app-modules/student-data-model/tests/Student/ListStudentsTest.php

```diff
@@ -91,13 +90,71 @@
 
     livewire(ListStudents::class)
         ->assertOk()
-        ->assertActionHidden(ImportAction::class);
+        ->assertActionHidden('importStudents');
 
     $user->givePermissionTo('student.import');
 
     livewire(ListStudents::class)
         ->assertOk()
-        ->assertActionVisible(ImportAction::class);
+        ->assertActionVisible('importStudents');
+});
+
+it('renders the importPrograms Action based on proper access', function () {
+    $user = User::factory()->licensed(Student::getLicenseType())->create();
+
+    $user->givePermissionTo('student.view-any');
+    $user->givePermissionTo('program.import');
```

Comment:
Let's fix all three of these tests for this.

We wouldn't need to revoke the import permission if we didn't give it here yet. Let's do so.